packages/web-components/fast-foundation/src/slider/slider.ts

```diff
@@ -113,6 +113,11 @@ export class Slider extends FormAssociated<HTMLInputElement>
     @observable
     public trackMinHeight: number = 0;
 
+    /**
+     * @internal
```

Comment:
I think we want this `@public` because the expectation is that can override this right? Also please add doc comments to describe the API and what purpose the property serves.

Reply:
thanks! updated to public and added doc comments